routes.go

```diff
@@ -221,43 +221,52 @@ func (a *routesCmd) call(c *cli.Context) error {
 }
 
 func routeWithFlags(c *cli.Context, rt *fnmodels.Route) {
-	if i := c.String("image"); i != "" {
-		rt.Image = i
+	if rt.Image == "" {
+		if i := c.String("image"); i != "" {
+			rt.Image = i
+		}
 	}
-
-	if f := c.String("format"); f != "" {
-		rt.Format = f
+	if rt.Format == "" {
+		if f := c.String("format"); f != "" {
+			rt.Format = f
+		}
 	}
-
-	if t := c.String("type"); t != "" {
-		rt.Type = t
+	if rt.Type == "" {
+		if t := c.String("type"); t != "" {
+			rt.Type = t
+		}
 	}
-
-	if m := c.Uint64("memory"); m > 0 {
-		rt.Memory = m
+	if rt.Memory == 0 {
+		if m := c.Uint64("memory"); m > 0 {
+			rt.Memory = m
+		}
 	}
-
-	if t := c.Duration("timeout"); t > 0 {
-		to := int32(t.Seconds())
-		rt.Timeout = &to
+	if rt.Timeout == nil {
+		if t := c.Duration("timeout"); t > 0 {
+			to := int32(t.Seconds())
+			rt.Timeout = &to
+		}
 	}
-
-	if t := c.Duration("idle-timeout"); t > 0 {
-		to := int32(t.Seconds())
-		rt.IDLETimeout = &to
+	if rt.IDLETimeout == nil {
+		if t := c.Duration("idle-timeout"); t > 0 {
```

Comment:
Let's just make these cli.IntFlags, get rid of any duration stuff.

Reply:
Done.